ctl/src/main.rs

```diff
@@ -0,0 +1,133 @@
+use raft_engine::Error;
+use raft_engine::Result;
+use std::path::PathBuf;
+use structopt::StructOpt;
```

Comment:
We usually group them in this order:
std::xx; newline; third_party::xx; newline; own_library::xx;

And please copy the copyright line to this file as well.

Reply:
This import order is recommended by the `cargo format`.  

> We usually group them in this order:
std::xx; newline; third_party::xx; newline; own_library::xx;

👌 :

> And please copy the copyright line to this file as well.
👌

Reply:
Yes, `format` will sort the imports if they are in the same group. That's why we explicit add newlines to create multiple groups.